aspnet/migration/rc1-to-rc2.rst

```diff
@@ -14,10 +14,53 @@ ASP.NET 5 RC1 apps were based on the .NET Execution Environment (DNX) and made u
 
 For information regarding a complete list of breaking changes in RC2 for ASP.NET Core, see the `ASP.NET Core Announcements page <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0-rc2>`_
 
+Changes in MVC
```

Comment:
Starting with MVC feels weird to me since it is at the top of the stack. Can we move this below "Namespace and package ID changes"?